src/retrieve-w3c-validation/node.ts

```diff
@@ -0,0 +1,22 @@
+// Imports
+import * as https from 'https';
+import { W3CCSSValidatorResponse } from '..';
+
+// Utility function for retrieving response from W3C CSS Validator in a Node.js environment
+const retrieveForNode = async (url: string): Promise<W3CCSSValidatorResponse['cssvalidation']> => {
```

Comment:
Let's use the word "in" instead of "for" here. Makes more sense to me.

```suggestion
const retrieveInNode = async (url: string): Promise<W3CCSSValidatorResponse['cssvalidation']> => {
```